src/schemata/runSettingsSchema.ts

```diff
@@ -0,0 +1,13 @@
+import * as S from '@effect/schema/Schema';
+
+export const runSettingsSchema = S.union(
+	S.struct({
+		dryRun: S.optional(S.literal(false)).withDefault(() => false),
```

Comment:
same here

Reply:
yes, that's a boolean literal always set to `false` in this case.